scalafix/rules/src/main/scala/fix/Collectionstrawman_v0.scala

```diff
@@ -135,10 +139,23 @@ case class Collectionstrawman_v0(index: SemanticdbIndex)
             ctx.replaceToken(close, ")")).getOrElse(Patch.empty)
     }.asPatch
 
+  def replaceTupleZipped(ctx: RuleCtx) =
+    ctx.tree.collect {
+      case tupleZipped(t @ Term.Select(Term.Tuple(args), _)) =>
+        val argTokens = args.flatMap(_.tokens).toSet
+        val removeTokensPatch = t.tokens.filter(!argTokens.contains(_)).map(ctx.removeToken).asPatch
+        val addTokensPatch = args.tail.map { arg =>
+          val tokens = arg.tokens
+          ctx.addLeft(tokens.head, ".lazyZip(") + ctx.addRight(tokens.last, ")")
+        }.asPatch
+        removeTokensPatch + addTokensPatch
+    }.asPatch
+
```

Comment:
I had to do this at the token level as suggested by @olafurpg due to a limitation in scalafix that is explaned here: https://gitter.im/scalacenter/scalafix?at=59ff60b35a1758ed0f90981b